docs/user-manual/_diagrams/client-classpath-jakarta.adoc

```diff
@@ -1 +1 @@
-This file will be generated/replaced in compile time by artemis-website
\ No newline at end of file
+This file will be generated/replaced in compile time by artemis-website
```

Comment:
Is this meant to be in the diagrams dir now?

Reply:
No. I nuked it.